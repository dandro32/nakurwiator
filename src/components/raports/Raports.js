import React from 'react';

const Raports = () => {
  return (<p>Raports</p>);
};

export default Raports;