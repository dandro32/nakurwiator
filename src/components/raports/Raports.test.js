import React from 'react';
import Raports from './Raports';
import TestRenderer from 'react-test-renderer';

jest.mock('../UI/dashboard/Dashboard', () => 'Dashboard component');

describe('Raports component tests', ()=> {
  let result;

  beforeEach(() => {
    result = TestRenderer.create(
      <Raports />
    ).toTree();

  });

  it('renders correctly', () => {
    expect(result).toMatchSnapshot();
  });
});