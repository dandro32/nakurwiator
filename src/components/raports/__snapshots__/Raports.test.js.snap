// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Raports component tests renders correctly 1`] = `
Object {
  "instance": null,
  "nodeType": "component",
  "props": Object {},
  "rendered": Object {
    "instance": null,
    "nodeType": "host",
    "props": Object {
      "cards": Array [
        Object {
          "desc": "Sprawdź raporty w formie timelinów",
          "href": "/raports/timeline",
          "hrefTitle": "Obczej",
          "imgSrc": "/img/raports_timeline.jpg",
          "title": "Timeliny",
        },
      ],
    },
    "rendered": Array [],
    "type": "Dashboard component",
  },
  "type": [Function],
}
`;
